pkg/controller/pxc/controller.go

```diff
@@ -273,6 +273,18 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 		nodeSet.Spec.Template.Annotations = make(map[string]string)
 	}
 	if cr.CompareVersionWith("1.1.0") >= 0 {
+		if r.serverVersion.Platform == api.PlatformOpenshift && cr.Spec.ProxySQL.ContainerSecurityContext != nil {
+			if *cr.Spec.ProxySQL.ContainerSecurityContext.Privileged {
+				nodeSet.Spec.Template.Annotations["openshift.io/scc"] = "privileged"
+				nodeSet.Spec.Template.Spec.Containers[0].SecurityContext = nil
+				nodeSet.Annotations = map[string]string{"openshift.io/scc": "privileged"}
+				nodeSet.Spec.Template.Spec.ServiceAccountName = "percona-operations"
```

Comment:
all should be tunable via annotations, PodSecurityContext in CR
maybe needed to add an option for nodeSet.Spec.Template.Spec.ServiceAccountName